### DISCUSSION TITLE
[HELP] Your account is fully operational.

### DISCUSSION THREAD

**Discussion by ABOhiccups** [mastodon/mastodon#30982] [Q&A] [answered]
Hello,
I’ve been running my own Mastodon server on YunoHost for several months. So, I’ve decided to uninstall YunoHost from my HomeLab and install Runtipi.

But, I forgot to backup my Mastodon server before wipe the entire SSD. After I installed Mastodon to Runtipi setup. I login to my Mastodon Account as owner. It told me my account is fully operational. I also get this yellow message.

Your application is pending review by our staff. This may take some time. You will receive an e-mail if your application is approved.

I just wanna know. Is all my following, followers, and posts transfer over to my new HomeLab setup since my old HomeLab setup is not online? Or is my Mastodon Account record from my old HomeLab setup is going be removed from Mastodon network?

My Mastodon server is
social.abohiccups.com

My Mastodon Account as owner is
@abohiccups@social.abohiccups.com

**Comment by dilroopg** [comment, marked as answer]
sudo docker exec -it -w /app/www mastodon bin/tootctl accounts approve "username" to approve your username and use tootctl in runtipi, idk about the rest

**Reply by dilroopg** [reply]
dont use the ""

**Reply by ABOhiccups** [reply]
Thank you so much!
